dominion/build.rs

```diff
@@ -0,0 +1,10 @@
+use scraper::*;
+
+fn main() {
```

Comment:
When does this run?

Reply:
`build.rs` is the default build file path. I should narrow down the `rerun-if-changed` variable though.

See https://doc.rust-lang.org/cargo/reference/build-scripts.html

Specifically:
> rerun-if-changed=PATH is a path to a file or directory which indicates that the build script should be re-run if it changes (detected by a more-recent last-modified timestamp on the file). Normally build scripts are re-run if any file inside the crate root changes, but this can be used to scope changes to just a small set of files. (If this path points to a directory the entire directory will not be traversed for changes -- only changes to the timestamp of the directory itself (which corresponds to some types of changes within the directory, depending on platform) will trigger a rebuild. To request a re-run on any changes within an entire directory, print a line for the directory and another line for everything inside it, recursively.) Note that if the build script itself (or one of its dependencies) changes, then it's rebuilt and rerun unconditionally, so cargo:rerun-if-changed=build.rs is almost always redundant (unless you want to ignore changes in all other files except for build.rs).